### ISSUE TITLE
[FTBFS] memtest86+

### ISSUE THREAD

**Issue by maxice8** [void-linux/void-packages#10359]
Package fails to build from source with:

```
ld --warn-constructors --warn-common -static -T memtest_shared.lds \
 -o memtest_shared head.o reloc.o main.o test.o init.o lib.o patn.o screen_buffer.o config.o cpuid.o linuxbios.o pci.o memsize.o spd.o error.o dmi.o controller.o smp.o vmem.o random.o && \
ld -shared -Bsymbolic -T memtest_shared.lds -o memtest_shared head.o reloc.o main.o test.o init.o lib.o patn.o screen_buffer.o config.o cpuid.o linuxbios.o pci.o memsize.o spd.o error.o dmi.o controller.o smp.o vmem.o random.o
ld: test.o: in function `L24':
test.c:(.text+0xb06): undefined reference to `__stack_chk_fail'
ld: test.o: in function `L7':
test.c:(.text+0xe89): undefined reference to `__stack_chk_fail'
ld: test.o: in function `L43':
test.c:(.text+0x1315): undefined reference to `__stack_chk_fail'
ld: test.o: in function `L73':
test.c:(.text+0x16d1): undefined reference to `__stack_chk_fail'
ld: test.o: in function `L123':
test.c:(.text+0x1ab7): undefined reference to `__stack_chk_fail'
make: *** [Makefile:29: memtest_shared] Error 1
```

Tried:

- Changing LD to CC
- Using -fno-stack-protector

**Comment by maxice8** [comment]
found out by #10358

**Comment by maxice8** [comment]
We believe this has been fixed in the latest version of memtest86+-5.01_5
